public/src/components/channelManagement/testVariantEditor.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { Tabs, Tab, Theme, makeStyles } from '@material-ui/core';
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  container: {
+    width: '100%',
+    paddingLeft: spacing(4),
+    paddingRight: spacing(4),
+
+    '& > * + *': { marginTop: spacing(2) },
+  },
+  splitContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  splitContent: {
+    width: 'calc(50% - 16px)',
+  },
+}));
+
+interface TestVariantEditorProps {
+  variantEditor: React.ReactElement;
+  variantPreview: React.ReactElement;
+}
+
+function TestVariantEditor({
+  variantEditor,
+  variantPreview,
+}: TestVariantEditorProps): React.ReactElement<TestVariantEditorProps> {
```

Comment:
This could maybe have a better name? It takes and editor + preview and renders the tabbed view. `EditorWithPreviewTabs`? or something to that affect?